activerecord/lib/active_record/base.rb

```diff
@@ -311,6 +311,7 @@ class Base
     include Attributes
     include Locking::Optimistic
     include Locking::Pessimistic
+    include OptimisticUpdate
```

Comment:
I initially created the module under `locking/`, but despite its similarities with `Locking::Optimistic`, I figured it wasn't _technically_ related to locking. Happy to hear suggestions though.